src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFile(fromFileName);
+        writeFile(content, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(
+                new FileReader(fromFileName))) {
+            String line = bufferedReader.readLine();
+            String[] split;
+            while (line != null) {
+                split = line.split(",");
+                switch (split[OPERATION_TYPE_INDEX]) {
+                    case "buy":
+                        buy += Integer.parseInt(split[AMMOUNT_INDEX]);
+                        break;
+                    default:
+                        supply += Integer.parseInt(split[AMMOUNT_INDEX]);
```

Comment:
```suggestion
                    case "buy":
                        buy += Integer.parseInt(split[AMMOUNT_INDEX]);
                        break;
                    case "supply":
                        supply += Integer.parseInt(split[AMMOUNT_INDEX]);
                        break;
```

default it for handle not expected values, but we expect to see "supply"